### ISSUE TITLE
Increased precision for probabilities close to 0 ?

### ISSUE THREAD

**Issue by symphorien** [nianzelee/ssatABC#23]
I am trying to use ssatABC on formulas with very low probablities and then make a quotient between them. For example the attached formula is expected to have a probability of about 2^-62 so ssatABC reports:
```
  > Satisfying probability: 0.000000e+00
```
but that leads me to do divisions of the form 0/0 which fails. Is there a way to get an increase precision?
[foo.sdimacs.txt](https://github.com/NTU-ALComLab/ssatABC/files/8484748/foo.sdimacs.txt)

**Comment by nianzelee** [comment]
No. The tool is written in C and uses `float` to store probabilities.
